iacrcc/tests/test28/main.tex

```diff
@@ -162,7 +162,7 @@ \section{Bibliography}
 \lipsum[1-20]
 
 % This sample uses bibtex rather than biblatex.
-\bibliography{template}
+% \bibliography{template}
```

Comment:
I'm not sure why this change was made, but it doesn't affect the test so it doesn't matter.

Reply:
On my end the template.bib file is not present in test28/ so I removed this.